### DISCUSSION TITLE
What will be the differences between transcribee and https://buzzcaptions.com?

### DISCUSSION THREAD

**Discussion by clstaudt** [bugbakery/transcribee#85] [Q&A] [answered]
The projects look superficially very similar, with [Buzz](https://buzzcaptions.com) being ready to use.

**Comment by anuejn** [comment, marked as answer]
Some things that I believe are different:

* Transcribee will be a collaborative tool (transcript correction a bit like etherpad / google docs)
* It will be a more "web-plattform-like" tool
* We will not focus on live transcriptions but on after the fact transcription
* All of the UI code (especially also for transcript-correction) will be open source
* We will support speaker-diarization and realignment of the transcript to the audio
